google/cloud/storage/BUILD.bazel

```diff
@@ -109,15 +107,19 @@ cc_library(
         "@platforms//os:windows": GOOGLE_CLOUD_STORAGE_WIN_DEFINES,
         "//conditions:default": [],
     }),
+    linkopts = select({
```

Comment:
Similar changes as in `google/cloud/BUILD.bazel`

Reply:
Done.